pkg/ovs/ovn-nb-logical_router_route.go

```diff
@@ -13,7 +13,12 @@ func (c OvnClient) GetLogicalRouterRouteByOpts(key, value string) ([]ovnnb.Logic
 	defer cancel()
 
 	api, err := c.ovnNbClient.WherePredict(ctx, func(r *ovnnb.LogicalRouterStaticRoute) bool {
-		return r.Options[key] == value
+		if len(r.Options) > 0 {
+			if v, ok := r.Options[key]; ok {
+				return v == value
+			}
+		}
+		return false
```

Comment:
```suggestion
		return len(r.Options) != 0 && r.Options[key] == value
```

Reply:
if key is not in r.Options , r.Options[key] will return ""  ,   and sometimes value may be "",    and then it will return true. but I want it return false